BUILD.bazel

```diff
@@ -179,3 +181,24 @@ alias(
     name = "urbit",
     actual = "//pkg/vere:urbit",
 )
+
+# gazelle:exclude external
+
+refresh_compile_commands(
+    name = "refresh_compile_commands",
+
+    # Specify the targets of interest.
+    # For example, specify a dict of targets and any flags required to build.
+    targets = [
+        "//pkg/ent",
+        "//pkg/noun",
+        "//pkg/ur",
+        "//pkg/urcrypt",
+        "//pkg/vere:urbit",
+    ],
+    # No need to add flags already in .bazelrc. They're automatically picked up.
+    # If you don't need flags, a list of targets is also okay, as is a single target string.
+    # Wildcard patterns, like //... for everything, *are* allowed here, just like a build.
+    # As are additional targets (+) and subtractions (-), like in bazel query https://docs.bazel.build/versions/main/query.html#expressions
+    # And if you're working on a header-only library, specify a test or binary target that compiles it.
+)
```

Comment:
This target should go in `//bazel` i.e. it should be `//bazel:refresh_compile_commands`, not `//:refresh_compile_commands`.